__tests__/app.test.js

```diff
@@ -379,3 +379,188 @@ describe("GET /api/users", () => {
       });
   });
 })
+
+describe("/api/articles?sort_by=query1&order=query2", () => {
+  test("200: Responds with an array of articles sorted by a column (query1) if there is no order", () => {
+    return request(app)
+      .get("/api/articles?sort_by=article_id")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(article).toMatchObject({
+            article_id: expect.any(Number),
+            title: expect.any(String),
+            topic: expect.any(String),
+            author: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            comment_count: expect.any(String),
+            article_img_url: expect.any(String)
+          });
```

Comment:
Because you have likely already tested the properties for your get articles endpoint - you could really just test here for the order!

Reply:
Thank you for highlighting that! Yes, it's not necessary in this describe block as it is done in another block. :)